app/models/ext_management_system.rb

```diff
@@ -473,7 +473,11 @@ def orchestrate_destroy
   before_destroy :assert_no_queues_present
 
   def assert_no_queues_present
-    throw(:abort) if MiqWorker.find_alive.where(:queue_name => queue_name).any?
+    if enabled?
+      orchestrate_destroy
```

Comment:
The method name doesn't sound like it would lead to the manager being destroyed...

Reply:
@bdunne I have a different approach now.